docs/README.md

```diff
@@ -61,6 +61,10 @@ Volunteers have translated many of our Contributing, How-to, and Code of Conduct
   - [How-to](HOWTO-it.md)
 - Japanese / 日本語
 - Kazakh / қазақша
+- Kannada / ಕನ್ನಡ
```

Comment:
should go above Kazakh (alphabetical order)

Reply:
ok wait

Reply:
i've corrected it
please check asap